pages/account/manage/event/[[...data]].js

```diff
@@ -69,6 +70,14 @@ export default function ManageEvent({ BASE_URL, event }) {
   const [endDate, setEndDate] = useState(
     event.date?.end && formatDate(event.date?.end)
   );
+  const DropDownOptions = ["Non-speaking","Speaking"];
+  const [userStatus, setuserStatus] = useState(event.userStatus || "");
+  const [speakingTopic, setspeakingTopic] = useState(event.speakingTopic || "Your speaking topic");
```

Comment:
Default should be an empty string, use html placeholder for suggestions

Reply:
Added default to be an empty string:

`const [speakingTopic, setspeakingTopic] = useState(event.speakingTopic || "");`

Added placeholder in the component:
```
 <Input
   type="text"
   name="topic"
   label="What topic are you speaking on ?"
   placeholder= "Your speaking topic"
   onChange={(e) => setspeakingTopic(e.target.value)}
   value={speakingTopic}
   required
   maxLength="256"
  />
```